metrics.go

```diff
@@ -0,0 +1,23 @@
+package gocron
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// JobStatus is the status of job run that should be collect
+// with metric.
+type JobStatus string
+
+// The different statuses of job that can be used.
+const (
+	Fail    JobStatus = "fail"
+	Success JobStatus = "success"
+)
+
+// Monitorer represents the interface to collect jobs metrics.
+type Monitorer interface {
```

Comment:
Can you make this just `Monitor` throughout?

```suggestion
// Monitor represents the interface to collect jobs metrics.
type Monitor interface {
```

Reply:
Sure, done